cmd/translatesfx/README.md

```diff
@@ -116,3 +130,343 @@ service:
       exporters:
       - signalfx
 ```
+
+## Translatable Features
+
+#### Monitors
+
+Smart Agent monitors are translated into Otel receivers and placed into the
+appropriate pipelines.
+
+For example, the Smart Agent monitor:
+
+```yaml
+monitors:
+  - type: vsphere
+    host: 1.2.3.4
+    username: user
+    password: abc123
+```
+
+would be translated into an Otel receiver and placed into the `metrics` pipeline:
+
+```yaml
+receivers:
+  smartagent/vsphere:
+    type: vsphere
+    host: 1.2.3.4
+    username: user
+    password: abc123
+service:
+  pipelines:
+    metrics:
+      receivers:
+        - smartagent/vsphere
+```
+
+#### Config Sources
+
+Smart Agent
+[remote configuration](https://docs.signalfx.com/en/latest/integrations/agent/remote-config.html)
+directives are translated into their corresponding Otel config sources.
+
+The types of config sources supported for translation are the following:
+
+###### Environment Variables
+
+Smart Agent environment variable interpolation in the form of `${VARNAME}` works similarly in Otel,
+so these are simply retained as is.
+
+Also supported is the more advanced form:
+
+```yaml
+signalFxAccessToken: {"#from": "env:SIGNALFX_ACCESS_TOKEN"}
+```
+
+which would be translated into:
+
+```yaml
+${SIGNALFX_ACCESS_TOKEN}
+```
+
+Note: support for default environment variable values is currently not suported but may be added in a future release.
```

Comment:
"**Note:** Support for for default environment variable values is currently not supported, but may be added in a future release."

Reply:
Fixed, with a minor alteration.